nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/coldstorage-contrib.xml

```diff
@@ -18,4 +18,24 @@
     </listener>
   </extension>
 
+  <!-- define the notification that will be responsible for sending the mails -->
+  <extension
+    target="org.nuxeo.ecm.platform.ec.notification.service.NotificationService"
+    point="notifications">
+
+    <notification name="ColdStorageContentAvailable" channel="email" enabled="true" availableIn="*"
+                  subject="Archive content available on '${docTitle}'"
+                  autoSubscribed="false" template="coldStorageContentAvailable"
+                  label="label.document.download">
```

Comment:
Aren't `enabled` and `autoSubscribed` set to their default values? In which case they can be removed for clarity.

Reply:
removed as they defined: `enable` to true and `autoSubscribed ` to false in the subscriber